ISA-XLSX.md

```diff
@@ -0,0 +1,734 @@
+| Assay Person Roles Term Accession Number | String | The accession number from the Term Source associated with the selected term. |
+| Assay Person Roles Term Source REF | String | Identifies the controlled vocabulary or ontology that this term comes from. The Source REF has to match one of the Term Source Names declared in the Ontology Source Reference section. |
+
+**Example**
+
+For example, the `ASSAY PERFORMERS` section of an ISA-XLSX `isa.assay.xlsx` file may look as follows:
+
+|                            |         |       | |
+|----------------------------|---------|-------|-|
+| ASSAY PERFORMERS |
+| Assay Person Last Name     | Zhang | Tzeng | Evans |
+| Assay Person First Name    | Ningning    | Shin-Cheng  | Bradley |
+| Assay Person Mid Initials  |       |       |
+| Assay Person Email         |  |       |
+| Assay Person Phone         |         |       |
+| Assay Person Fax           |         |       |
+| Assay Person Address       | St. Louis, Missouri 63132, USA | St. Louis, Missouri 63132, USA | St. Louis, Missouri 63132, USA |
+| Assay Person Affiliation   | Donald Danforth Plant Science Center | Donald Danforth Plant Science Center | Donald Danforth Plant Science Center |
+| Assay Person Roles         | Investigator | Laboratory Technologist | Laboratory Technologist |
+| Assay Person Roles Term Accession Number | http://purl.obolibrary.org/obo/NCIT_C25936 | http://purl.obolibrary.org/obo/NCIT_C51830 | http://purl.obolibrary.org/obo/NCIT_C51830 |
+| Assay Person Roles Term Source REF       | NCIT | NCIT | NCIT |
+
+
+# Annotation Table sheets
+
+In the `Annotation Table sheets`, column headers MUST have the first letter of each word in upper case, with the exception of the referencing label (REF).
+
+The content of the annotation table MUST be placed in an `xlsx table` whose name starts with `annotationTable`. Each sheet MUST contain at most one such annotation table. Only cells inside this table are considered as part of the formatted metadata.
+
+`Annotation Table sheets` are structured with fields organized on a per-row basis. The first row MUST be used for column headers. Each body row is an implementation of a `Process` node.
+
+## Inputs and Outputs
+
+Each annotation table sheet MUST contain an `Input` and an `Output` column, which denote the Input and Output node of the `Process` node respectively. They MUST be formatted in the pattern `Input [<InputNodeType>]`.
+
+
+A `Source` MUST be indicated with the node type `Source Name`. `Sources` MUST not be used as `Output` nodes.
```

Comment:
Does this also imply that there MUST be a `Source Name` (somewhere in the ARC), i.e. a `Sample Name` MUST not exist without a Source towards that Sample. 

Or did I misunderstand that isa tables are changed towards allowing `Input [Sample Name]`?

Reply:
Yes, exactly. With this change, we want to make the annotation more explicit. 

Instead of
```
-----Sheet1-----|------Sheet2------|
Source -> Sample = Source -> Sample
```
we now have
```
---Sheet1---|---Sheet2---|
Source -> Sample -> Sample
```
The second entity in the example above was in actuality a `Sample`, but ambiguously annotated as a `Source` in the second sheet. Now instead, we annotate it as what it is, a `Sample` in both Sheets.

Reply:
Perfect, but this only answers half the question. 

Can there be a sheet in any isa.study or isa.assay starting with a Sample, that does not have a source?

Reply:
I would say yes, and therefore not add any constraint.

To my understanding, `Source` is only a further specification of a `Sample`.